help/actions/_posts/2015-02-09-export.md

```diff
@@ -140,10 +148,22 @@ The output field specifiers are added to the beginning of the custom format as d
 
     This is a special format directive. It must be the first command specified in a format string, and there can be multiple *%Z* directives at the beginning of the string. While the *Z* tag produces no output, it allows you to specify the value of certain variables with corresponding parameters. It has the following syntax: **%Zcommand:parameter**. The parameter can be a number, a word, or a string, depending on the command. If it is a string, it must be enclosed in double quotes. The available commands are: 
 
-    * **Encoding:word** This specifies the encoding of the output. Available encoding parameters are: 
-        * **unicode** (default): UTF-8 encoding of all unicode characters. 
+    There are two encoding options: global and field. As the name suggests specifying global encoding applies the encoding to the entire citiation, all output field. Please see further down for explanation of field encoding.
+    * **Encoding:word** Available global encoding parameters are: 
+        * **unicode** (default): UTF-8 encoding of all unicode characters. Note also for backward compatibility **UTF-8** option for word is also supported.
         * **html**: This encoding converts characters in URLs to hex notation, and & to &amp;. Outside URLs, the characters &, <, >, and " are converted to &amp;, &lt;, &gt;, and &quot;, respectively. 
         * **latex**:  This encoding converts all special Tex characters into their Tex escape sequences. For instance '\\' is converted to '$\backslash $', '$' is converted into '\\$', '^' is converted into '\\^{}', etc.
-    * **Linelength:number** This specifies the output line length (default is 80 characters). Use a line length of 0 to avoid line wraps. 
+        * **csv**: The encoding creates csv file format including header line if not specified. 
+    * **Linelength:number** This specifies the output line length (default is 0 meaning no line wraps). 
     * **Header:&ldquo;string&rdquo;** This specifies a header that will be output once before the data records. 
     * **Footer:&ldquo;string&rdquo;** This specifies a footer that will be output once after all the data records are output. 
+    * **Markup:word**	This specifies how to process HTML markup found in input fields (such as ≤SUB>). Available options are:
+        * **keep**: keep the markup (default)
+        * **strip**: remove the markup
+    * **EOL:&ldquo;string&rdquo;** This specifies the character to be printed at the end of a record (newline by default). 
+    * **AuthorSep:&ldquo;string&rdquo;** This specifies the string to be used when concatenating the names of authors (", " by default). 
+
+* **Field encoding** is applied to each individual field. Available field encoding characters are:
+    * **>**: html encoding. This encoding converts characters in URLs to hex notation, and & to &amp;. Outside URLs, the characters &, <, >, and " are converted to &amp;, &lt;, &gt;, and &quot;, respectively. 
```

Comment:
* same issue as for the global HTML encoding - `&amp;`, etc., are all getting converted into actual characters, so wrap them in backticks

Reply:
thank you so much for cutting this so here is what we have now
```
html: This encoding converts characters in URLs to hex notation, and & to &amp;. Outside URLs, the characters &, <, >, and " are converted to &amp;, &lt;, &gt;, and &quot;, respectively.
```